app/src/main/java/org/wikipedia/descriptions/DescriptionEditBottomBarView.kt

```diff
@@ -0,0 +1,84 @@
+package org.wikipedia.descriptions
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.View
+import androidx.constraintlayout.widget.ConstraintLayout
+import kotlinx.android.synthetic.main.view_description_edit_read_article_bar.view.*
+import org.wikipedia.R
+import org.wikipedia.suggestededits.SuggestedEditsSummary
+import org.wikipedia.util.DimenUtil
+import org.wikipedia.util.L10nUtil.setConditionalLayoutDirection
+import org.wikipedia.util.ResourceUtil
+import org.wikipedia.util.StringUtil
+import org.wikipedia.views.ViewUtil
+
+
+class DescriptionEditBottomBarView @JvmOverloads constructor(
+        context: Context, attrs: AttributeSet? = null, defStyle: Int = 0) : ConstraintLayout(context, attrs, defStyle) {
+
+    init {
+        inflate(context, R.layout.view_description_edit_read_article_bar, this)
+        hide()
+    }
+
+    fun show() {
+        visibility = VISIBLE
+    }
+
+    fun hide() {
+        visibility = GONE
+    }
+
+    fun setSummary(summary: SuggestedEditsSummary) {
```

Comment:
I think it would be nice if we keep the same method and add an `InvokeSource` to do future logic of setting views.

Reply:
Makes sense